demo/c/test_demo.py

```diff
@@ -0,0 +1,225 @@
+#
+# Copyright 2018-2021 Picovoice Inc.
```

Comment:
The way we usually do the copyright dates is:
`{year created} - {year last modified}`

In the case of `year created == year last modified` it's just `year created`.
```suggestion
# Copyright 2021 Picovoice Inc.
```